src/widgets/heightandweight/heightandweight.resource.ts

```diff
@@ -16,7 +16,7 @@ function getDimensionsObservations(patientId: string) {
     `${fhirConfig.baseUrl}/Observation?subject:Patient=${patientId}&code=${WEIGHT_CONCEPT},${HEIGHT_CONCEPT}`
   ).pipe(
     map(({ data }) => data["entry"]),
-    map((entries: []) => entries.map((entry: any) => entry.resource)),
+    map(entries => entries.map(entry => entry.resource)),
```

Comment:
What I meant was that we should type the response to know exactly what `data` is. Then we know exactly whats on `data` and if we would type `entry.resourc` (missing 'e') the compiler would complain.

Right now its just empty and we need to do `data["entry"]` to go on. With `any`, which means no compiler assistance here.

Reply:
:+1:  thats makes it clean